src/wallet/node_rpc_proxy.cpp

```diff
@@ -42,28 +42,27 @@ NodeRPCProxy::NodeRPCProxy(epee::net_utils::http::http_simple_client &http_clien
   : m_http_client(http_client)
   , m_daemon_rpc_mutex(mutex)
   , m_height(0)
-  , m_height_time(0)
   , m_earliest_height()
   , m_dynamic_per_kb_fee_estimate(0)
   , m_dynamic_per_kb_fee_estimate_cached_height(0)
   , m_dynamic_per_kb_fee_estimate_grace_blocks(0)
   , m_rpc_version(0)
   , m_target_height(0)
-  , m_target_height_time(0)
+  , m_block_size_limit(0)
+  , m_get_info_time(0)
 {}
 
 void NodeRPCProxy::invalidate()
 {
   m_height = 0;
-  m_height_time = 0;
   for (size_t n = 0; n < 256; ++n)
     m_earliest_height[n] = 0;
   m_dynamic_per_kb_fee_estimate = 0;
   m_dynamic_per_kb_fee_estimate_cached_height = 0;
   m_dynamic_per_kb_fee_estimate_grace_blocks = 0;
   m_rpc_version = 0;
   m_target_height = 0;
-  m_target_height_time = 0;
+  m_get_info_time = 0;
```

Comment:
Maybe also do `m_block_size_limit = 0;` just in case? Though I guess technically not needed.

Reply:
No, it should be as you said.